advocacy_docs/pg_extensions/otel/configuring.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Configuring EDB OTEL
+navTitle: Configuring
+---
+
+The following configuration options are available with edb_otel.
+
+## Permissions
+
+No permissions changes are required for edb_otel.
+
+## Prerequisites
+
+Before using edb_otel to export metrics, you must set up an OpenTelemetry collector endpoint that edb_otel can communicate with.  For detailed information on how set up a quick test of the Opentelemetry endpoint using docker, see [OTLP Exporter Example](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/examples/otlp).
+
+## GUCs
+
+The following GUC variables are available in edb_otel:
+
+| Variable                       | Description | Unit | Default | Minimum value |                                                                                                                                                                                                                                                |
+|--------------------------------|-------------|------|---------|---------------|
+| `edb_otel.edb_otel_endpoint`   | Specifies the OTEL endpoint URL | | | |
+| `edb_otel.interval`            | Interval between two consecutive exports. It translates to the opentelemetry metric reader parameter `exportIntervalMillis`.| Milliseconds | 2 seconds | 1 second|  
+| `edb_otel.timeout`             | Specifies how long the export can run before it's cancelled. It translates to the opentelemetry metric reader parameter `exportTimeoutMillis`.| Milliseconds | 500 milliseconds | 100 milliseconds|
+| `edb_otel.worker_nap_time`     | Duration of background worker sleep time before it checks for metrics in the queue and processes them. | Milliseconds | 2 seconds | 1 second |
+| `edb_otel.metrics_queue_size`  | Interval between two consecutive exports. This is the maximum size of the metrics queue. If the queue reaches this size, any subsequent metrics are dropped until the queue is cleared while processing the metrics inside it.| MB | 100 | 100|
+
+## Configuring to be used by a SQL interface
+
+To use the edb_otel extension from a SQL interface, use the following steps:
+
+In postgresql.conf:
+
+1. Set the parameter `edb_otel.edb_otel_endpoint` to point to the collector endpoint.  Example: 
+```bash
+edb_otel.edb_otel_endpoint='localhost:4317'
+```
+2. Add `edb_otel` to the shared_preload_libraries. Example: 
+```bash
+shared_preload_libraries = '$libdir/dbms_pipe,$libdir/edb_gen,$libdir/dbms_aq,$libdir/edb_otel'
+```
+## Configuring to be used within C code
+
+To use the edb_otel extension from your C code, use the following steps:
+
+1. Ensure the `edb_otel_ext.h` file is within reach of your extension at build time. The file is placed in `$prefix/include/server/extension/edb_otel/` when the edb_otel dev/devel package is installed. Your Makefile must add `pg_config --includedir-server` to its include search paths, so your extension can use it as:
```

Comment:
Here you start a numbered list that is not continued. You could continue the numbered list or remove the number here so it is only text. I think it is better to continue the list.